src/link/style.scss

```diff
@@ -10,6 +10,38 @@
 	}
 }
 
+.wikipediapreview-edit-preview-popover {
+	.components-popover__content {
+		background: unset;
+		border: unset;
+		box-shadow: none;
+	}
+	.wikipediapreview {
+		box-shadow: none;
+	}
+	.wikipediapreview-body::after {
+		width: 350px;
+	}
+}
+
+.wikipediapreview-edit-preview-container {
+	display: flex;
+	align-items: center;
+}
```

Comment:
the `width: 350px in :: after` doesn't work in my env, but it works on the container, see the screenshot

| current | width in container |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/2560096/132375860-0ac1fec9-76a0-40de-997e-c24f89623747.png) | ![image](https://user-images.githubusercontent.com/2560096/132375736-192031ed-e579-42c2-bb46-3b6c32174fcd.png) |

Reply:
Hmm you're right, I saw the same faulty `::after` styling when adding the RTL support too. It's probably best to hide it with `content: unset` then (https://github.com/wikimedia/wikipediapreview-wordpress/pull/30/commits/32622fde28f871e18a4b23c29f4cb04ab8d93300)